etc/bytedump.sh

```diff
@@ -13,17 +13,18 @@ Abort.
 EOF
 } | python3 -c '
 import os, sys
-needle = b"\nCOQTOP_CRAP_ENDS_HERE\n"
+sep = os.linesep.encode("ascii")
+needle = b"COQTOP_CRAP_ENDS_HERE" + sep
 waiting_on_coqbug_15373 = True
 b = b""
 while True:
 	r = os.read(0, 1)
 	if not r:
-		sys.exit(waiting_on_coqbug_15373 + 2*(not b.endswith(b"\n")))
+		sys.exit(waiting_on_coqbug_15373 + 2*(not b.endswith(sep)))
```

Comment:
What about adding
```suggestion
		exitcode = waiting_on_coqbug_15373 + 2*(not b.endswith(sep))
		if exitcode:
		  print(f'Error: read {b!r}', file=sys.stderr)
		  sys.exit(exitcode)
```
for debugging purposes?